### PR TITLE
feat: UI optimizations for avatar, personal addresses and addNewAddresses modal

### PR DESCRIPTION
**Pull request by Esther-Lita**
- Created a new `ExternalEmailModal` component for adding external email addresses to the organization
- Updated `AddEmailModal` component to open the `ExternalEmailModal` when selecting 'Add External Email Instead'
- Removed the obsolete external email page and transferred its functionalities to the new `ExternalEmailModal` component
- Added state management for modals in the organization mail addresses page
-Resizing avatar on account creation
-Properly aligned personal addresses

fixes issues: ENG-51 ENG-69 ENG-47

---

## What does this PR do?

Fixes # (issue)

_If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists_

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by Esther-Lita** [comment]
* **#621** <a href="https://app.graphite.dev/github/pr/un/inbox/621?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Esther-Lita and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortcode]/settings/org/mail/addresses/_components/add-address-modal.tsx [hunk 1]
```diff
@@ -30,11 +30,11 @@ import { Button } from '@/src/components/shadcn-ui/button';
 import { Switch } from '@/src/components/shadcn-ui/switch';
 import { Input } from '@/src/components/shadcn-ui/input';
 import { zodResolver } from '@hookform/resolvers/zod';
+import { DialogClose } from '@radix-ui/react-dialog';
```
Comment (BlankParticle) [inline apps/web/src/app/[orgShortcode]/settings/org/mail/addresses/_components/add-address-modal.tsx:33]: import from `@/src/components/shadcn-ui/switch` from `@radix-ui/*`

File: apps/web/src/app/[orgShortcode]/settings/org/mail/addresses/_components/add-external-email-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+    encryption: z.enum(['none', 'ssl', 'tls', 'starttls']),
+    authMethod: z.enum(['plain', 'login'])
+  }),
+  deliversTo: z.object({
+    users: z.string().array(),
+    teams: z.string().array()
+  })
+});
+
+export default function ExternalEmailModal({
```
Comment (BlankParticle) [inline apps/web/src/app/[orgShortcode]/settings/org/mail/addresses/_components/add-external-email-modal.tsx:55]: don't use `default` export, it has issues with names while importing

**Review by BlankParticle — CHANGES_REQUESTED** [review]
Also check CI and fix those errors

**Review by BlankParticle — APPROVED** [review]

**Pull request merged**